2017-01-30-openwhisk-integration-with-serverless

```diff
@@ -0,0 +1,44 @@
+---
+title: Introducing OpenWhisk Integration with the Serverless Framework
```

Comment:
Great feedback @pmuens! Thank you! I'll keep it as "OpenWhisk" since it's the most common reference.